### PR TITLE
Merger

### PR DESCRIPTION
**Pull request by Divinity-dev**
I'm now merging the two branches of this repository.

### CODE REVIEW

**Comment by Divinity-dev** [comment]
Merging my branches

**Comment by Divinity-dev** [comment]
Close

**Comment by Divinity-dev** [comment]
Reopening

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (mckent05) [inline test.md]: - According to the project requirements, this file should be ignored by `git`, please remove this file from your repo and ensure it is ignored by your `gitignore` file 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Divinity-dev,

# STATUS: CHANGE REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Good job setting up your repo 👏 


## Required Changes ♻️
- Please note that in your future microverse projects, when working with the GitHub flow, the `main` branch should be your base branch, so you would be required to merge all PR's into the `main` branch, currently you have the `copy` branch as your base branch. Please fix this 👍 👍 
_Check the comments under the review._

- Also it is important that your PR is professional, hence you need to have a descriptive PR title and summary that briefly describes the changes you have made in a PR.
- To edit your PR title, open your PR and click on the 'edit' button, reference the picture attached below:
rgwrgwg
- To edit your PR sumary, open your PR and click on the button, reference the picture attached below:
fwhaeth
- In your subsequent project in Microverse, it is important that all linter checks pass in order for your code to be reviewed, remember that linters help us ensure that we write code that is clean and easy to maintain, please make sure all your linter checks are passing 🙏 

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**